./pseudo_intrinsics.h:110:25: error: use of 'long long' with '__vector' requires VSX support (available on POWER7 or later) to be enabled

Build on ppc64 (big-endian) fails without adding -mvsx to CFLAGS:
```
cc -DAC_BUILT -DJOHN_ALTIVEC -c -m64 -O2 -pipe  -fstack-protector-strong -fno-strict-aliasing  -I/usr/local/include -I/usr/include -DARCH_LITTLE_ENDIAN=0 -DJOHN_SYSTEMWIDE -DJOHN_SYSTEMWIDE_EXEC='"/usr/local/bin"' -DJOHN_SYSTEMWIDE_HOME='"/usr/local/share/john"'  -DJOHN_SYSTEMWIDE=1 -Wall -Wno-tautological-constant-out-of-range-compare -fno-omit-frame-pointer -Wno-deprecated-declarations -Wformat-extra-args -Wunused-but-set-variable -Qunused-arguments -std=gnu89 -Wdate-time -D__BSD_VISIBLE -I/usr/include -fopenmp  -D_THREAD_SAFE -pthread -I/usr/local/include -DCL_SILENCE_DEPRECATION -funroll-loops -O1 john.c
In file included from john.c:109:
In file included from ./dynamic.h:30:
In file included from ./simd-intrinsics.h:19:
./pseudo_intrinsics.h:110:25: error: use of 'long long' with '__vector' requires VSX support (available on POWER7 or later) to be enabled
typedef vector unsigned long long vtype64;
                        ^
1 error generated.
```

This is FreeBSD 14.0-CURRENT with LLVM 14.0.5.

--- a/src/john.c
+++ b/src/john.c
@@ -106,7 +106,6 @@ static int john_omp_threads_new;
 #include "subsets.h"
 #include "external.h"
 #include "batch.h"
-#include "dynamic.h"
 #include "dynamic_compiler.h"
 #include "fake_salts.h"
 #include "listconf.h"
@@ -141,6 +140,8 @@ static int john_omp_threads_new;
 #endif
 #include "omp_autotune.h"
 
+extern int dynamic_Register_formats(struct fmt_main **ptr);
+	
 #if CPU_DETECT
 extern int CPU_detect(void);
 extern char CPU_req_name[];

